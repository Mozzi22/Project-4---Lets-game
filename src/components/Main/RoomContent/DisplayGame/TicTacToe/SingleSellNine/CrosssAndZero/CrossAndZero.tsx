import React from 'react';
import { StCrossAndZero } from './styled'
interface IProps {
    figure: string | null;
}

const CrossAndZero: React.FC<IProps> = ({ figure }) => {
    return (
        <StCrossAndZero
        >
            {figure}
        </StCrossAndZero>
    );
};

export default CrossAndZero;
