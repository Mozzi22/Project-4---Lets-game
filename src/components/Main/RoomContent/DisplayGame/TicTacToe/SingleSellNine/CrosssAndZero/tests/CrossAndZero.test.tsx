import React from 'react';
import CrossAndZero from '../CrossAndZero';
import { mount, shallow } from 'enzyme';

describe('CrossAndZero', () => {
    const props = {
        figure: 'X'
    }
    it('Should match snapshot', () => {
        const component = shallow(<CrossAndZero {...props}/>);
        expect(component.html()).toMatchSnapshot();
    });
    it('should render StSingleSellNine', () => {
        const component = mount(<CrossAndZero {...props}/>);
        expect(component.find('styled__StCrossAndZero')).toHaveLength(1);
    });
});
