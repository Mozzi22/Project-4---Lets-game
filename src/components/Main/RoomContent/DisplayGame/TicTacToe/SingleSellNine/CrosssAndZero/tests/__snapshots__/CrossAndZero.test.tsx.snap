// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CrossAndZero Should match snapshot 1`] = `"<div class=\\"styled__StCrossAndZero-sc-1e5txb5-0 bWPZNh\\">X</div>"`;
